open-company/open-company-web
# Readonly report

card: https://trello.com/c/iFXgXv09

test:
- add some reports for company XXX
- visit /companies/XXX
- select a report, you should see the readonly report
- visit /companies/edit/XXX/{YEAR}/{QUORTER} for the previous report, you should be able to edit data
